common/persistence/sql/execution.go

```diff
@@ -678,7 +679,7 @@ func (m *sqlExecutionStore) GetCurrentExecution(
 	})
 	if err != nil {
 		if err == sql.ErrNoRows {
-			return nil, serviceerror.NewNotFound(err.Error())
+			return nil, serviceerror.NewNotFound("workflow not found")
 		}
```

Comment:
This was the only place I could see where we return ErrNoRows to the caller, instead of a custom error.